### PR TITLE
YDSK: harrowing_swarm.txt + support

### PR DESCRIPTION
**Pull request by Northmoc**
Replaces #6371

### CODE REVIEW

File: forge-gui/res/cardsfolder/h/harrowing_swarm.txt [hunk 1]
```diff
@@ -0,0 +1,10 @@
+Name:Harrowing Swarm
+ManaCost:1 G
+Types:Sorcery
+A:SP$ ManifestDread | SubAbility$ DBAnimate | SpellDescription$ Manifest dread.
+SVar:DBAnimate:DB$ AnimateAll | ValidCards$ Creature.faceDown+YouCtrl | Zone$ Battlefield | staticAbilities$ ReduceCost | Triggers$ TurnedFaceUp | Duration$ Permanent | StackDescription$ SpellDescription | SpellDescription$ Then each face-down creature you control gains "This permanent costs {2} less to turn face up" and "When this permanent is turned face up, if it's a creature, put a +1/+1 counter on it."
+SVar:ReduceCost:Mode$ ReduceCost | Amount$ 2 | Type$ TurnFaceUp | ValidCard$ Card.Self | Activator$ You | EffectZone$ All | Description$ This permanent costs {2} less to turn face up.
+SVar:TurnedFaceUp:Mode$ TurnFaceUp | ValidCard$ Creature.Self | Execute$ TrigPump | TriggerZones$ Battlefield | TriggerDescription$ When this permanent is turned face up, if it's a creature, put a +1/+1 counter on it. | Description$ When this permanent is turned face up, if it's a creature, put a +1/+1 counter on it.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/h/harrowing_swarm.txt:7]: Description$?

Comment (tool4ever) [inline forge-gui/res/cardsfolder/h/harrowing_swarm.txt:7]: intervening if not handled :p

File: forge-game/src/main/java/forge/game/cost/CostAdjustment.java [hunk 2]
```diff
@@ -526,64 +526,71 @@ private static boolean checkRequirement(final SpellAbility sa, final StaticAbili
...
+                        return false;
+                    }
+                    if (st.hasParam("FirstForetell") && activator.getNumForetoldThisTurn() > 0) {
+                        return false;
+                    }
                 }
-                if (st.hasParam("FirstForetell") && activator.getNumForetoldThisTurn() > 0) {
-                    return false;
+                case "TurnFaceUp" -> {
+                    if (!sa.isTurnFaceUp()) return false;
```
Comment (tool4ever) [inline forge-game/src/main/java/forge/game/cost/CostAdjustment.java:590]: why new type?
I'd rather see you use `ValidSpell$ Static.isTurnFaceUp`

File: forge-gui/res/cardsfolder/h/harrowing_swarm.txt (hunk 1 above)
Comment (Northmoc) [reply forge-gui/res/cardsfolder/h/harrowing_swarm.txt:7]: Wouldn't this be covered by `ValidCard$ Creature.Self` ?

Comment (Northmoc) [reply forge-gui/res/cardsfolder/h/harrowing_swarm.txt:7]: Ha ha probably copypasta

File: forge-game/src/main/java/forge/game/cost/CostAdjustment.java (hunk 2 above)
Comment (Northmoc) [reply forge-game/src/main/java/forge/game/cost/CostAdjustment.java:590]: I didn't realize that would work 😆

File: forge-gui/res/cardsfolder/h/harrowing_swarm.txt (hunk 1 above)
Comment (tool4ever) [reply forge-gui/res/cardsfolder/h/harrowing_swarm.txt:7]: no, that's only checked for the trigger event

File: forge-game/src/main/java/forge/game/cost/CostAdjustment.java (hunk 2 above)
Comment (tool4ever) [reply forge-game/src/main/java/forge/game/cost/CostAdjustment.java]: syntax still wrong...

**Review by tool4ever — APPROVED** [review]

**Pull request merged**